HW05B_script.R

```diff
@@ -0,0 +1,69 @@
+#HW05 Part B
+#Author: Hannah Farrell
+#Date: 8/3/2020
+
+#load appropriate packages
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+library(stringr)
+
+#load in the data
+AUC1 <- read.csv("./CTRP_files/AUC_1.csv", header = TRUE)
+AUC2 <- read.csv("./CTRP_files/AUC_2.csv", header = TRUE)
+cell_line <- read.csv("./CTRP_files/cancer_cell_line_info.csv", header = TRUE)
+compound <- read.csv("./CTRP_files/compound_info.csv", header = TRUE)
+experiment <- read.csv("./CTRP_files/Experiment_info.csv", header = TRUE)
+
+#merge the AUC data
+merged_AUC <- AUC1 %>%
+  full_join(AUC2)
+#add in the 3 other csvs based on matching column names
+full_data_set <- merged_AUC %>%
+  full_join(experiment, by = c("experiment_id" = "expt_id")) %>%
+  full_join(cell_line, by = "master_ccl_id") %>%
+  full_join(compound, by = "master_cpd_id")
+
```

Comment:
Although I understand that the idea was to highlight the cancer types with lowest AUC values (thus the use of top_n when filtering), I think it would be more informative to make the figure with all cancer types (there were not too many). This way it would have been easier to visualize the cancer types with the lowest values (identified by top_n) but also the cancer types with lowest values on average. And in general, visualize all the data, not only extreme values. In sum, I would have left out the top_n when filtering.